config/application.rb

```diff
@@ -33,6 +33,9 @@ class Application < Rails::Application
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
     config.time_zone = 'London'
 
+    # Minutes the EA area lookup job should run for
+    config.ea_area_lookup_run_for = ENV["EA_AREA_LOOKUP_RUN_FOR"] || 60
```

Comment:
See previous comment about the names of the area across WEX and FRAE

```suggestion
    config.area_lookup_run_for = ENV["AREA_LOOKUP_RUN_FOR"] || 60
```